quandl/model/dataset.py

```diff
@@ -0,0 +1,23 @@
+from quandl.operations.get import GetOperation
+from quandl.operations.list import ListOperation
+from quandl.util import Util
+from .model_base import ModelBase
+from .data import Data
+from .database import Database
+
+
+class Dataset(GetOperation, ListOperation, ModelBase):
+
+    @classmethod
+    def get_path(cls):
+        return "%s/metadata" % cls.default_path()
+
+    def data(self, **options):
+        # default order to ascending, and respect whatever user passes in
+        params = {'database_code': self.database_code,
```

Comment:
needs more consideration, there is arguments for either case, and ruby client already allows user to override